lib/saluki-components/src/sources/otlp/metrics/cache.rs

```diff
@@ -197,8 +221,8 @@ mod tests {
         message: &'static str,
     }
 
-    #[test]
-    fn test_monotonic_diff_unknown_start() {
+    #[tokio::test]
+    async fn test_monotonic_diff_unknown_start() {
```

Comment:
Do these actually need to be async? My recollection was that `CacheBuilder::for_tests` should give us a cache with no expiration interval and no telemetry... which should lead us to not spawning any background tasks.

Reply:
You're right, updated.